metadeploy/api/models.py

```diff
@@ -939,6 +940,23 @@ def complete(self, config):
         self.save()
         async_to_sync(notify_org_finished)(self)
 
+    def get_refreshed_org_config(self):
+        org_config = OrgConfig(self.config, self.plan.org_config_name)
+        info = jwt_session(
+            settings.CONNECTED_APP_CLIENT_ID,
+            settings.CONNECTED_APP_CLIENT_KEY,
```

Comment:
FWIW, if these settings happened to come from environment variables names SFDX_CLIENT_ID and SFDX_HUB_KEY, then all we would need to do is `org_config.refresh_oauth_token(None)` and CumulusCI would do the rest of this based on the presence of those environment variables. (Without them, that method uses a different approach for refreshing the token which won't work here.)

Reply:
@davisagli do you think that's a worthwhile change? These env vars are only used in a couple places in the code, env files, and CI config. I can take a shot at it on a separate PR.

Reply:
@jerivas It's pretty low priority. You could add it to a list of cleanup tasks to do if you have time while waiting for UX / frontend at some point.

Reply:
I've added it to the ice box